.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,82 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: dropdown
+    id: command
+    attributes:
+      label: Which command you find issue with?
+      multiple: false
+      options:
+        - create
+        - generate
+        - run
+        - generate-certs
+        - logs
+        - sessionlog
+        - cacheinfo
+        - shutdown
+    validations:
+      required: true
+  - type: dropdown
+    id: platforms
+    attributes:
+      label: On what platforms did you experience the issue?
+      multiple: true
+      options:
+        - Windows
+        - MacOS
+        - Linux
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: What version are you using?
+      placeholder: 'package : version'
+    validations:
+      required: true
+  - type: textarea
```

Comment:
Would it make sense to add as well the mode? 

- Development
- Staging
- Production